pkgs/desktops/nixos-cinnamon/default.nix

```diff
@@ -0,0 +1,12 @@
+self: super:
+
+{
+  cinnamon = super.callPackage ./cinnamon { };
+  cinnamon-desktop = super.callPackage ./cinnamon-desktop { };
+  cinnamon-menus = super.callPackage ./cinnamon-menus { };
+  cinnamon-session = super.callPackage ./cinnamon-session { gconf = super.gnome2.GConf; };
```

Comment:
Does this still require GConf? It has been deprecated over 8 years ago.

Reply:
Apparently. I've had to add it again.